### DISCUSSION TITLE
login page css modification

### DISCUSSION THREAD

**Discussion by DIVYANSGupta** [requarks/wiki#6886] [Help / Questions] [answered]
hey @NGPixel, I want to customize the css of "**Login**" page, How can I do it ? Which ***.scss** is it referring to ?

**Comment by NGPixel** [comment, marked as answer]
https://github.com/requarks/wiki/blob/main/client/components/login.vue#L694

**Comment by rcronin** [comment]
@NGPixel - can this be modified on theme code injection text box by chance?

<img width="1424" alt="image" src="https://github.com/requarks/wiki/assets/11710708/f2ed7804-be8c-4da9-8ec7-10ad4b52c923">

**Reply by NGPixel** [reply]
No. It doesn't apply to the login page.
